R/agyw-integration.R

```diff
@@ -0,0 +1,1517 @@
+#' Format naomi outputs for PSE tool
+#'
+#' @param outputs Naomi output
+#' @param options Naomi model options.
+#'
+#'
+#' @return Naomi indicators formatted for the AGYW workbook.
+#'
+#' @export
```

Comment:
Does this actually need to be exported? Will you use it from anywhere other than Naomi?

Reply:
Thanks - have removed.